lighty-modules/lighty-ovsdb-sb/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2018 Pantheon Technologies s.r.o. All Rights Reserved.
+
+  This Source Code Form is subject to the terms of the LIGHTY.IO LICENSE,
+  version 1.1. If a copy of the license was not distributed with this file,
+  You can obtain one at https://lighty.io/license/1.1/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.lighty.core</groupId>
+        <artifactId>lighty-parent</artifactId>
+        <version>9.2.1-SNAPSHOT</version>
+        <relativePath>../../lighty-core/lighty-parent</relativePath>
+    </parent>
+
+    <groupId>io.lighty.modules</groupId>
+    <artifactId>lighty-ovsdb-sb</artifactId>
+
+    <properties>
+        <jaxrs.version>2.1</jaxrs.version>
+        <jersey.version>2.26</jersey.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.lighty.core</groupId>
+            <artifactId>lighty-controller</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
please put ovsdb artefacts into lighty-core/dependency-versions/pom.xml

Reply:
check if it's as you wanted